knowledge-base/sign-pdf-with-signature-widget.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Signing a PDF Document with a SignatureWidget
+description: This article provides instructions on how to digitally sign an existing PDF document using RadPdfProcessing.
+type: how-to
+page_title: Signing a PDF Document with a SignatureWidget
+slug: sign-pdf-with-signature-widget
+tags: pdf, sign, widget
+res_type: kb
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 2024.1.124 | RadPdfProcessing |[Desislava Yordanova](https://www.telerik.com/blogs/author/desislava-yordanova)| 
+
+## Description
+
+This article shows how to digitally sign an existing PDF document that already contains a SignatureField.
```

Comment:
Add link to SignatureField - https://docs.telerik.com/devtools/document-processing/libraries/radpdfprocessing/model/interactive-forms/form-fields/signaturefield